keras_cv/models/models_test.py

```diff
@@ -156,6 +158,18 @@ def _test_model_can_be_used_as_backbone(self, app, last_dim, args):
         model = keras.Model(inputs=inputs, outputs=[backbone_output])
         model.compile()
 
+    def _test_model_serialization(self, app, _, args, save_format, filename):
```

Comment:
This code should already be available in #1367

Reply:
+1 -- @soma2000-lang could you please merge from master@head and then re-push? otherwise lgtm.